src/schemathesis/models.py

```diff
@@ -147,6 +147,7 @@ class TestResult:
     checks: List[Check] = attr.ib(factory=list)  # pragma: no mutate
     errors: List[Tuple[Exception, Optional[Case]]] = attr.ib(factory=list)  # pragma: no mutate
     is_errored: bool = attr.ib(default=False)  # pragma: no mutate
+    seed: Optional[int] = attr.ib(default=None)  # pragma: no mutate
```

Comment:
I am not sure if it should be optional since there is always some seed in hypothesis

Reply:
If there is a hypothesis test. In case of invalid schema there is no test.

Reply:
aha, got it